module/zfs/vdev_label.c

```diff
@@ -359,9 +359,6 @@ root_vdev_actions_getprogress(vdev_t *vd, nvlist_t *nvl)
 {
 	spa_t *spa = vd->vdev_spa;
 
-	if (vd != spa->spa_root_vdev)
```

Comment:
[1] I wanted to make the fix as simple as possible for easy review.  I agree that it would be better to have a single  `ZPOOL_CONFIG_SCAN_STATS` in the root vdev nvlist, rather than a copy in each vdev.  I dunno why it was done that way.  Maybe someone can refactor that code that in another PR.

[2] Yea, I think something like `spa_pool_actions_getprogress(spa_t *spa, nvlist_t *nvl)` makes the most sense.  I'll make the change

Reply:
Ok, I renamed it `vdev_actions_getprogress()` in my latest push.